package/utils/utils.go

```diff
@@ -87,6 +90,13 @@ func GetOffset(str string) (int, error) {
 	return int(offset), nil
 }
 
+func ValidateUserRole(role string, acceptedRoles []models.UserRole) error {
```

Comment:
I don't like name of this function. My initial thought was that we are validating if user role is correct itself. Maybe ValidateRoleAccess will be better name? Or something in that manner?